src/fsharp/FSharp.Compiler.Server.Shared/AssemblyInfo.fs

```diff
@@ -1,10 +1,15 @@
 // Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 namespace Microsoft.FSharp
+
 open System.Reflection
+open Microsoft.VisualStudio.Shell
+
 [<assembly:System.Runtime.CompilerServices.InternalsVisibleTo("Fsi, PublicKey=002400000480000094000000060200000024000052534131000400000100010007D1FA57C4AED9F0A32E84AA0FAEFD0DE9E8FD6AEC8F87FB03766C834C99921EB23BE79AD9D5DCC1DD9AD236132102900B723CF980957FC4E177108FC607774F29E8320E92EA05ECE4E821C0A5EFE8F1645C4C0C93C1AB99285D622CAA652C1DFAD63D745D6F2DE5F17E5EAF0FC4963D261C8A12436518206DC093344D5AD293")>]
 [<assembly:System.Runtime.CompilerServices.InternalsVisibleTo("FsiAnyCPU, PublicKey=002400000480000094000000060200000024000052534131000400000100010007D1FA57C4AED9F0A32E84AA0FAEFD0DE9E8FD6AEC8F87FB03766C834C99921EB23BE79AD9D5DCC1DD9AD236132102900B723CF980957FC4E177108FC607774F29E8320E92EA05ECE4E821C0A5EFE8F1645C4C0C93C1AB99285D622CAA652C1DFAD63D745D6F2DE5F17E5EAF0FC4963D261C8A12436518206DC093344D5AD293")>]
 [<assembly:System.Runtime.CompilerServices.InternalsVisibleTo("FSharp.VS.FSI, PublicKey=002400000480000094000000060200000024000052534131000400000100010007D1FA57C4AED9F0A32E84AA0FAEFD0DE9E8FD6AEC8F87FB03766C834C99921EB23BE79AD9D5DCC1DD9AD236132102900B723CF980957FC4E177108FC607774F29E8320E92EA05ECE4E821C0A5EFE8F1645C4C0C93C1AB99285D622CAA652C1DFAD63D745D6F2DE5F17E5EAF0FC4963D261C8A12436518206DC093344D5AD293")>]
 
+[<assembly: ProvideCodeBase(CodeBase = @"$PackageFolder$\FSharp.Compiler.Server.Shared.dll")>]
```

Comment:
Well, today I learned what ProvideCodeBaseAttribute does... For this interested: https://msdn.microsoft.com/en-us/library/microsoft.visualstudio.shell.providecodebaseattribute.aspx